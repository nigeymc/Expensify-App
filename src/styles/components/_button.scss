.button {
    background-color: $blue;
    border: none;
    color: #fff;
    display: inline-block;
    font-size: $font-size-large;
    font-weight: 300;
    line-height: 1;
    outline: none;
    padding: $s-size;
    text-decoration: none;
}

.button--link {
    background-color: transparent;
}

.button--secondary {
    background-color: #888;
}