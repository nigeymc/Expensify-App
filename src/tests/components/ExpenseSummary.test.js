import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpenseSummary';

test('should render ExpenseSummary with expenses', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={23} expenseTotal={2348574083} />);
    expect(wrapper).toMatchSnapshot();
});

test('should render ExpenseSummary with 1 expense', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={1} expenseTotal={234} />);
    expect(wrapper).toMatchSnapshot();
});