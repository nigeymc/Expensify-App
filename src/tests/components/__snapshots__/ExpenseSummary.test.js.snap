// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $0.00
  </h1>
</div>
`;

exports[`should render ExpenseSummary with expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $0.00
  </h1>
</div>
`;
